packages/@aws-cdk/aws-codepipeline/lib/action.ts

```diff
@@ -116,6 +118,8 @@ export interface CommonActionProps {
    * @see https://docs.aws.amazon.com/codepipeline/latest/userguide/reference-pipeline-structure.html
    */
   readonly runOrder?: number;
+
+  readonly roleArn?: string;
```

Comment:
Docs

Reply:
This field should actually not be there. Removed.